From: Ben Hutchings <benh@debian.org>
Date: Fri, 31 Dec 2021 14:59:23 +0100
Subject: [klibc] Kbuild: Include provided KCFLAGS in KLIBCCFLAGS

If KCFLAGS is provided through the environment or make variables, use
it in addition to all the options we need.

---
--- a/scripts/Kbuild.klibc
+++ b/scripts/Kbuild.klibc
@@ -78,7 +78,7 @@ KLIBCOPTFLAGS     :=
 KLIBCSHAREDFLAGS  :=
 KLIBCBITSIZE      :=
 KLIBCLDFLAGS      :=
-KLIBCCFLAGS       :=
+KLIBCCFLAGS       := $(KCFLAGS)
 
 # Defaults for arch to override
 KLIBCARCHINCFLAGS =
